src/coreclr/pal/src/arch/amd64/context2.S

```diff
@@ -81,6 +81,7 @@ LEAF_END CONTEXT_CaptureContext, _TEXT
 
 LEAF_ENTRY RtlCaptureContext, _TEXT
     mov     DWORD PTR [rdi + CONTEXT_ContextFlags], (CONTEXT_AMD64 | CONTEXT_FULL | CONTEXT_SEGMENTS)
+    mov     QWORD PTR [rdi + CONTEXT_XStateFeaturesMask], 0
     jmp     C_FUNC(CONTEXT_CaptureContext)
 LEAF_END RtlCaptureContext, _TEXT
```

Comment:
Just something else to note...

It seems that `CONTEXT_CaptureContext` doesn't currently and, as far as I can tell, has never captured the YMM state. Is this expected? Should it be capturing that state as part of the thread suspend/restore logic here?

Reply:
@tannergooding the CONTEXT_CaptureContext is used only by the debugger code on Unix. It is not used for thread suspension. I am not sure if debugger stuff cares about the YMM registers or not.